src/whatsnew/2.1.rst

```diff
@@ -46,6 +46,15 @@ Upgrade Notes
   had been unfortunately misnamed, and has now been updated to behave as the
   name would suggest. Both are documented in the shipped ``default.ini`` file.
 
+* :ghissue:`914`: Certain critical config sections are blacklisted from being
```

Comment:
Might be nice to mention these sections here, if they're not too esoteric.

Reply:
They're pretty esoteric - no one should be accessing these over the HTTP API.